apps/src/weblab/WebLab.js

```diff
@@ -135,6 +142,18 @@ WebLab.prototype.init = function (config) {
     isProjectLevel: !!config.level.isProjectLevel,
   });
 
+  function onAddFileHTML() {
+    this.brambleHost.addFileHTML();
+  }
+
+  function onAddFileCSS() {
+    this.brambleHost.addFileCSS();
+  }
+
+  function onAddFileImage() {
+    dashboard.assets.showAssetManager(null, 'image', this.loadCurrentAssets.bind(this), !this.studioApp_.reduxStore.getState().pageConstants.is13Plus);
```

Comment:
It'd be nice to extract a `this.pageConstants()` helper on this class, just to simplify that last argument.